Doc/Tutorial/chapter_seq_objects.tex

```diff
@@ -686,24 +664,16 @@ \section{Comparing Seq objects}
 Sequence comparison is actually a very complicated topic, and there is no easy
 way to decide if two sequences are equal. The basic problem is the meaning of
 the letters in a sequence are context dependent - the letter ``A'' could be part
-of a DNA, RNA or protein sequence. Biopython uses alphabet objects as part of
-each \verb|Seq| object to try to capture this information - so comparing two
-\verb|Seq| objects could mean considering both the sequence strings \emph{and}
-the alphabets.
-
-For example, you might argue that the two DNA \verb|Seq| objects
-\texttt{Seq("ACGT", IUPAC.unambiguous\_dna)} and
-\texttt{Seq("ACGT", IUPAC.ambiguous\_dna)} should be equal, even though
-they do have different alphabets. Depending on the context this could be
-important.
-
-This gets worse -- suppose you think \texttt{Seq("ACGT",
-IUPAC.unambiguous\_dna)} and \texttt{Seq("ACGT")} (i.e. the default generic
-alphabet) should be equal. Then, logically, \texttt{Seq("ACGT", IUPAC.protein)}
-and \texttt{Seq("ACGT")} should also be equal. Now, in logic if $A=B$ and
-$B=C$, by transitivity we expect $A=C$. So for logical consistency we'd
-require \texttt{Seq("ACGT", IUPAC.unambiguous\_dna)} and \texttt{Seq("ACGT",
-IUPAC.protein)} to be equal -- which most people would agree is just not right.
+of a DNA, RNA or protein sequence. Biopython can track the molecule type, so
+comparing two \verb|Seq| objects could mean considering this too.
+
+Support you argue \texttt{Seq("ACGT", generic\_dna)`` and \texttt{Seq("ACGT")}
+should be equal (i.e. comparing DNA and the default generic alphabet).
+Then, logically, \texttt{Seq("ACGT", generic_protein)} and \texttt{Seq("ACGT")}
+should also be equal. Now, in logic if $A=B$ and $B=C$, by transitivity we
+expect $A=C$. So for logical consistency we'd require
+\texttt{Seq("ACGT", generic\_dna)} and \texttt{Seq("ACGT", generic\_protein)}
+to be equal -- which most people would agree is just not right.
```

Comment:
exactly...

Reply:
I'll fix the typo added in the first word (Support/suppose) in editing the old paragraphs.